BO4E/ENUM/WahlrechtPrognosegrundlage.cs

```diff
@@ -0,0 +1,33 @@
+namespace BO4E.ENUM
+{
+    /// <summary>
+    /// Wahlrecht der Prognosegrundlage der Marktlokation
+    /// </summary>
```

Comment:
wäre super service, wenn du hier noch hinschreiben würdest (in `<remark>`), was der edifact qualifier ist.

Reply:
ja dann schau mal :-)